src/components/Trends/GetHypedTrends.js

```diff
@@ -2,18 +2,34 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { compose, pure } from 'recompose'
 import * as actions from './actions.js'
+import { simpleSort } from '../../utils/sortMethods'
 
 class GetHypedTrends extends React.Component {
   render () {
-    const { render, ...rest } = this.props
-    return render(rest)
+    const { render, items, ...rest } = this.props
+    const trends = items.slice(-1)
+    const props = {
+      ...rest,
+      items: trends.length > 0 ? [sortByHype(trends)] : trends
```

Comment:
I don't understand this case. If trends.length <= 0, trends === []
why did you write ? ... : trends ?

Also you return only one array from items. What were items before we returned?
Can we create text for this.

Reply:
lat question: https://github.com/santiment/app/blob/feature/hype-score-sort/src/components/Trends/TrendsTable/TrendsTables.js

this component expect array ("trends" prop), but it really not nesessary, as I understand. Should I rewrite this logic here?
inside array: datetime, topWords, __typename fileds